libgraph/include/katana/PropertyGraph.h

```diff
@@ -883,19 +883,19 @@ class KATANA_EXPORT PropertyGraph {
   Result<void> DeleteEdgeIndex(const std::string& column_name);
 
   // Returns the list of node indexes.
-  const std::vector<std::unique_ptr<PropertyIndex<GraphTopology::Node>>>&
+  const std::vector<std::unique_ptr<EntityIndex<GraphTopology::Node>>>&
   node_indexes() const {
     return node_indexes_;
   }
 
   // Returns the list of edge indexes.
-  const std::vector<std::unique_ptr<PropertyIndex<GraphTopology::Edge>>>&
+  const std::vector<std::unique_ptr<EntityIndex<GraphTopology::Edge>>>&
   edge_indexes() const {
     return edge_indexes_;
   }
 
   // Returns true of an index exists for the named property
-  bool HasNodePropertyIndex(const std::string& property_name) const {
+  bool HasNodeIndex(const std::string& property_name) const {
```

Comment:
@arthurp We need this function exposed to Python users. I can do that or maybe you are doing that already? (Not sure whether this is the reason you are renaming things to make things easier)

Currently, python users seem not able to check the existence of
the node index. Without this, we cannot correctly test double-create
or double-delete cases properly for indexing in pytest. Turns out one specific order
of running index-related tests worked for a while without proper graph
state management. @roshandathathri

Reply:
The rename is because "PropertyIndex" now refers to an index into a property array, since that will not be the same as the node or edge ID (soon).

This feature is not exposed in Python yet. I'm not even sure I have a request for it. Please create a JIRA task and assign it to me if you need it. I'm working on PropertyGraph wrappers now, so I might be able to make it happen as part of that work.

Reply:
@arthurp 
Just created a JIRA ticket. The tests of interest here are here:
https://github.com/KatanaGraph/katana-enterprise/blob/master/python/test/remote_graph_test.py#L283

Talking to @arekay  to confirm whether this is truly wanted. At least it is a bug to me when refactoring the RunQuery/LoadGraph within QueryClient.

@roshandathathri